robot_specific_config/abb_irb2600_support/urdf/irb2600_12_185_macro.xacro

```diff
@@ -0,0 +1,183 @@
+      	<origin xyz="-0.15 0 -0.445"/>
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/visual/link_2.stl"/>
+        </geometry>
+        <xacro:material_abb_yellow />
+      </visual>
+    </link>
+    <link name="${prefix}link_3">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/collision/link_3.stl"/>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/visual/link_3.stl"/>
+        </geometry>
+        <xacro:material_abb_yellow />
+      </visual>
+    </link>
+    <link name="${prefix}link_4">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/collision/link_4.stl"/>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/visual/link_4.stl"/>
+        </geometry>
+        <xacro:material_abb_yellow />
+      </visual>
+    </link>
+    <link name="${prefix}link_5">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/collision/link_5.stl"/>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/visual/link_5.stl"/>
+        </geometry>
+        <xacro:material_abb_yellow />
+      </visual>
+    </link>
+    <link name="${prefix}link_6">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/collision/link_6.stl"/>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb2600_support/meshes/irb2600_12_185/visual/link_6.stl"/>
+        </geometry>
+        <xacro:material_abb_yellow />
+      </visual>
+    </link>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint type="revolute" name="${prefix}joint_1">
+      <origin xyz="0 0 0.445" rpy="0 0 0"/>
+      <axis xyz="0 0 1"/>
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1"/>
+      <!-- See note 2 in package.xml about effort limits and dynamics values -->
+      <limit lower="-3.14159" upper="3.14159" effort="0" velocity="3.054"/>
+      <dynamics damping="50.0" friction="1.0"/>
+    </joint>
+    <joint type="revolute" name="${prefix}joint_2">
+      <origin xyz="0.15 0 0" rpy="0 0 0"/>
+      <axis xyz="0 1 0"/>
+      <parent link="${prefix}link_1"/>
+      <child link="${prefix}link_2"/>
+      <!-- See note 2 in package.xml about effort limits and dynamics values -->
+      <limit lower="-2.705" upper="1.658" effort="0" velocity="3.054"/>
```

Comment:
Same comments regarding joint limits as the `irb2600_12_185` file above.

Reply:
fixed in commit [e633967](https://github.com/PickNikRobotics/abb_ros2/pull/53/commits/e6339671fa35b2cd25fc848a7da47aac8ea0ae33)